source/NVDAObjects/UIA/__init__.py

```diff
@@ -2438,3 +2437,20 @@ def _get_value(self) -> typing.Optional[str]:
 		else:
 			val = ((val - minVal) / (maxVal - minVal)) * 100.0
 		return f"{round(val)}%"
+
+
+class EditUIA(EditBase, UIA):
+	"""Overlay class for Edit and RichEdit controls."""
+
+	def _get_name(self):
+		name = super().name
+		if name == "RichEdit Control":

```

Comment:
Have you tested that this name is not localized on non-English versions of Windows? If it tunrs out that it indeed is localized, perhaps we can just retrieve the IAccessible name with oleacc.

Reply:
I have tested on a Dutch version of Windows that the name is indeed not localized. I considered using oleacc but that this would probably be quicker. Never mind though, only writing this down for future reference.